pkg/setting/setting_openfeature.go

```diff
@@ -0,0 +1,41 @@
+package setting
+
+const (
+	StaticProviderType = "static"
+	GOFFProviderType   = "goff"
+)
+
+type OpenFeatureSettings struct {
+	Subpath string // to configure path for open feature api endpoints
+
+	ProviderType string
+	URL          string
+	TargetingKey string
+	ContextAttrs map[string]any
+}
+
+func (cfg *Cfg) readOpenFeatureSettings() {
+	cfg.OpenFeature = OpenFeatureSettings{}
+
+	config := cfg.Raw.Section("feature_toggles.openfeature")
+
+	cfg.OpenFeature.Subpath = config.Key("subpath").MustString("")
```

Comment:
I wonder if we should use `server.serve_from_sub_path` and `server.root_url` instead?

Reply:
In such a case we don't need anything extra: if subpath is specified, endpoints would be served from subpath

Reply:
ahh! I didn't realize this, thank you 🙇‍♂️